blueprints/admin/routes.py

```
@@ -0,0 +1,253 @@
+            password=throwaway_password,
+            is_superadmin=False,
+        )
+
+        try:
+            db.session.add(new_admin)
+            db.session.commit()
+            flash("New admin added successfully.", "success")
+            return redirect(url_for("admin.manage_admins"))
+        except Exception:
+            db.session.rollback()
+            flash(f"A fatal error occurred while trying to add a new admin", "danger")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)